### ISSUE TITLE
A11y_AzureDataStudioExtensionAzureSQLMigration_Migratetoazuresql_DatabasesforAssessment_ScreenReader: ORCA screen reader is announcing "Learn more" link as button.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23854]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG4.1.2; #FTP; #SH-AzureDataStudioExtensionAzureSQLMigration-Linux-Jul23; #SQL Migration Control Plane; #Linux;

### Environment Details:
Application Name: Azure Data Studio extension 'Azure SQL Migration'
Operating System: Ubuntu 20.04.2 LTS
Azure data studio 1.44.1
Azure SQL Migration v1.4.8
Screen Reader: ORCA

### Repro Steps:

1. Install Azure data studio application and open it
2. TAB till Extension press ENTER key and install Azure SQL Migration extension.
3. TAB till Respective server and right click.
4. TAB till manage and press ENTER key.
5. TAB till Azure SQL Migration and press ENTER key.
6. TAB till "migrate to azure sql" and press ENTER.
7. Select start new session and fill all the data and press ENTER on next button. Start ORCA screen reader.
8. Databases for assessment page will open. TAB to "Learn More link" and observe the issue.

### Actual Result:
ORCA screen reader is announcing "Learn more" link as button.

### Expected Result:
ORCA screen reader should announce "Learn more" link as "Learn more link".

### User Impact:
The screen reader users will get incorrect information and will get confused whether the control is link or button.

### Attachment:

https://github.com/microsoft/azuredatastudio/assets/90901519/ef161d97-adb3-47b5-8f52-326992392ea2

**Comment by kisantia** [comment]
@PoojaNamde a [similar bug](https://github.com/microsoft/azuredatastudio/issues/22690) is a sev3. Can this be downgraded to sev3?
